clang/include/clang/Driver/Options.td

```diff
@@ -3330,10 +3331,10 @@ defm relaxed_template_template_args : BoolFOption<"relaxed-template-template-arg
           "Enable C++17 relaxed template template argument matching">,
   NegFlag<SetFalse>>;
 defm sized_deallocation : BoolFOption<"sized-deallocation",
-  LangOpts<"SizedDeallocation">, DefaultFalse,
+  LangOpts<"SizedDeallocation">, Default<cpp14.KeyPath>,
   PosFlag<SetTrue, [], [ClangOption, CC1Option],
```

Comment:
CC1Option should be removed from PosFlag, and the driver should be changed to not pass -fsized-deallocation to cc1.

Reply:
Thanks, I think I have fixed it now. :-)